converters/lance-image-dataset-converter/convert-any-image-dataset-to-lance.py

```diff
@@ -0,0 +1,106 @@
+import os
+import argparse
+import pandas as pd
+import pyarrow as pa
+import lance
+import time
+from tqdm import tqdm
+
+def process_images(images_folder, data_type):
+    # Define schema for RecordBatch
+    schema = pa.schema([('image', pa.binary()), 
+                        ('filename', pa.string()), 
+                        ('category', pa.string()), 
+                        ('data_type', pa.string())])
+
+    # Iterate over the categories within each data type
+    category_folder = os.path.join(images_folder, data_type)
```

Comment:
If I understand correctly, `category` means the image class, right? If that is the case, let's use that name throughout.